core/src/main/java/com/graphhopper/routing/ev/MaxSpeed.java

```diff
@@ -33,6 +33,10 @@
      * The speed value used for road sections without known speed limit.
      */
     public static final double UNSET_SPEED = Double.POSITIVE_INFINITY;
+    /**
+     * The lowest speed value which can be stored.
+     */
+    public static final double MINIMUM_STOREABLE_SPEED = 5;
```

Comment:
This is not that easy and dependent on how the EncodedValue for the speed is configured. So we can easily store 0.1km/h with enough bits...

Reply:
I'm not sure how (or if?) we should deal with unstoreable values. If we know that we can't store a certain value, we could use alternative tags or fallback values.

Reply:
The tag extraction methods shouldn't care about it and always return precise numbers. Instead this issue can be handled in OSMMaxSpeedParser dependent on the configuration of the EncodedValue. As this is an important issue it is worth a separate issue.